Formula/gdal-20.rb

```diff
@@ -275,13 +272,8 @@ def install
     system "make", "install"
 
     inreplace "swig/python/setup.cfg", /#(.*_dirs)/, "\\1"
+    inreplace "swig/python/setup.cfg", "include_dirs = ../../port:../../gcore:../../alg:../../ogr/","include_dirs = ../../port:../../gcore:../../alg:../../ogr/:../../apps/"
```

Comment:
Thanks, I knew there was better code that I did not know.

> On May 16, 2016, at 5:27 PM, Larry Shaffer <notifications@github.com> wrote:
> 
> In Formula/gdal-20.rb <https://github.com/OSGeo/homebrew-osgeo4mac/pull/140#discussion_r63449012>:
> 
> > @@ -275,13 +272,8 @@ def install
> >      system "make", "install"
> >  
> >      inreplace "swig/python/setup.cfg", /#(.*_dirs)/, "\\1"
> > +    inreplace "swig/python/setup.cfg", "include_dirs = ../../port:../../gcore:../../alg:../../ogr/","include_dirs = ../../port:../../gcore:../../alg:../../ogr/:../../apps/"
> I think this will do the same
> 
> inreplace "swig/python/setup.cfg", /(include_dirs.+$)/, "\\1:../../apps/"
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/OSGeo/homebrew-osgeo4mac/pull/140/files/04542794328a926bccb8fbd096e4bd595f0da89f#r63449012>